specification/cognitiveservices/Language.QuestionAnswering-authoring/routes.tsp

```diff
@@ -151,6 +153,47 @@ interface QuestionAnsweringProjects {
     ErrorResponse
   >;
 
+  /**
+   * Import project assets from file.
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "endpoint has already been GA'd"
+  #suppress "@azure-tools/typespec-azure-core/byos" "method overload `import` already support byos with fileUri"
+  #suppress "@azure-tools/typespec-autorest/unsupported-multipart-type" "method overload `import` already support byos with fileUri"
+  @added(Versions.v2025_05_15_preview)
+  @sharedRoute
+  @route("/authoring/query-knowledgebases/projects/{projectName}/:import")
+  @pollingOperation(QuestionAnsweringProjects.getImportStatus)
+  importFromFile is Foundations.LongRunningOperation<
```

Comment:
In case of single file support, keep it same. In case of multiple file support rename it to importFromFiles